src/basic_recipes/axis.jl

```diff
@@ -125,7 +123,7 @@ end
         frame = Theme(
             linecolor = (grid_color, grid_color, grid_color),
             linewidth = (grid_thickness, grid_thickness, grid_thickness),
-            axiscolor = (darktext, darktext, darktext),
```

Comment:
This has no 2d equivalent?

Reply:
ah, I'm actually also not using it in the code :D this needs a clean up at some point ;)

Reply:
You are using it, I am starting to understand what's going on. Cleanup and unifying a bit 2d and 3d interface would be good, I'll keep my eyes on it and contribute.